nixops/index.tt

```diff
@@ -43,8 +43,8 @@ should be instantiated (in this case, as VirtualBox virtual machines
 running on your desktop). You can then run:</p>
```

Comment:
This diff is nothing but updating `<span class="muted">$</span> my command` to `<span class="command">my command</span>`